tests/test_geometry.py

```diff
@@ -1541,6 +1541,22 @@ def test_lonlat_bounds():
     with pytest.raises(ValueError):
         geometry.lonlat_bounds(geometry.box(0, 0, 1, 1, None))
 
+    multi = {
```

Comment:
I don't know why this change is here. I think it's from a different (already merged) PR via rebase weirdness.